lms/services/jstor.py

```diff
@@ -0,0 +1,28 @@
+from lms.views.helpers import via_url
+
+
+class JSTORService:
+    def __init__(self, settings):
+        self._settings = settings
+
+    @property
+    def enabled(self):
+        return self._settings.get("enabled", False)
+
+    @staticmethod
+    def via_url(request, document_url):
+        return via_url(
+            request,
+            document_url,
+            content_type="pdf",
+            options={"jstor.ip": request.registry.settings["jstor_ip"]},
+        )
+
+
+def factory(_context, request):
+    # application_instance.settings is an ApplicationSettings which overwrites `get`
+    # convert to a dict to be able to access ["jstor"] securely
```

Comment:
`ApplicationSettings.get` takes a group and key, expecting always nested values.